File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/Java/1_Array.js
public class ArrayJava {
    public static void main(String[] args) {
        //KHAI bao ARRAY TINH
        int[] arr = {1,2,3};
        //Voi Java k can q.ly so luong p.tu
        System.out.print("arr: ");
        for (int i=0; i< arr.length; i++){
            System.out.print(arr[i] + " ");
        }
        System.out.println();

        //KHAI BAO ARRAY DONG
        int[] pArr = new int[3]; //value ban dau luon = 0 -> ko can memset nhu C++
        //Voi Java k can q.ly so luong p.tu
        System.out.print("pArr: ");
        for (int i=0; i<pArr.length; i++){
            System.out.print(pArr[i] + " ");

        }
    }
}

File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/C++/1_Array.cpp
#include <iostream>
#include <cstring> //thu vin cua memset
using namespace std;

int main()
{
    // khai bao mang tinh (vung cap phat nho stack)
    int arr[] = {1,2,3};
    int size = sizeof(arr)/sizeof(int); //q.ly so luong p.tu
    cout << "arr: ";
    for (int i=0; i<size; i++) //xai for duyet cac p.tu trong array
    {
        cout << arr[i] << " ";
    }
    cout << endl;


    // khai bao mang dong (vung cap phat nho heep)
    int *pArr = new int[3]; //dung toan tu new
    int pSize = 3; //q.ly so luong p.tu
		//chon 1 vung nho de cap phat (memset dong bo all cac vung nho ve 0)
    memset(pArr, 0x00, pSize * sizeof(int));
    cout << "pArr: ";
    for (int i=0; i<pSize; i++) //xai for duyet cac p.tu trong array
    {
        cout << pArr[i] << " ";
    }
    cout << endl;
    delete[] pArr;
    return 0;
}